src/SFA.DAS.Notifications.Infrastructure/NotifyEmailService/NotifyHttpClientWrapper.cs

```diff
@@ -51,7 +51,15 @@ public NotifyHttpClientWrapper(IConfigurationService configurationService)
                     Content = stringContent
                 });
 
-                EnsureSuccessfulResponse(response);
+                try
+                {
+                    EnsureSuccessfulResponse(response);
```

Comment:
If an exception is thrown here is it not bubbled up to one of these?

* https://github.com/SkillsFundingAgency/das-notifications/blob/master/src/SFA.DAS.Notifications.Infrastructure/ExecutionPolicies/SendEmailExecutionPolicy.cs#L28
* https://github.com/SkillsFundingAgency/das-notifications/blob/master/src/SFA.DAS.Notifications.Worker/MessageHandlers/QueuedNotificationMessageHandler.cs#L52
* https://github.com/SkillsFundingAgency/das-notifications/blob/master/src/SFA.DAS.Notifications.Worker/WorkerRole.cs#L101